### ISSUE TITLE
Is Jire still active ?

### ISSUE THREAD

**Issue by COLATHOS** [Jire/Charlatano#455]
Hi Jire. Are you still active ? Or is it useless to write you ? A new release 15 could be expected ? Thanks to reply

**Comment by CyAn1D3** [comment]
He is Retired, He went to the Himalayas.

**Comment by anurag01021996** [comment]
Yes.. @Cyan1D3

**Comment by Jire** [comment]
I'll update tonight which should properly fix a few things.

**Comment by COLATHOS** [comment]
Hi Jire. Nice to meet you. Very happy to read your post ! 
I was afraid you were away from Charlatano : - ) 
Greetings from PARIS - France

**Comment by attitude12** [comment]
Did you update it, @Jire ? Do we need to rebuild or what? Thanks. :)

**Comment by COLATHOS** [comment]
@Jire: I use Windows 7 professional. I cannot fully use your programme (ESP not working, AIM not working, ..). Only the triggerbot is working, but only time to time. Hitchance at 100% ? Anyway, your hack looks good and secure. 
Could you confirm that I absolutely need Windows 10 and its update ? 
And test in windowed borderless if fullscreen unsuccefull ?
Thanks

**Comment by COLATHOS** [comment]
Hi attitude. Nice to meet you too

**Comment by attitude12** [comment]
@COLATHOS Yeah, this issue was discussed many times. If you want it to be fully functional, you need latest Window 10.

**Comment by COLATHOS** [comment]
Yes. sorry.  I just read all the former threads (until 90 days ago) ..... discussed many times indeed. You right. Only Windows 10.  Thanks. I have to get Windows 10
